doc/wintabs.txt

```diff
@@ -218,23 +218,31 @@ default: ' '
                 they have the same length as separators, and are ascii 
                 characters.
 
-*g:wintabs_ui_name_format*
+*g:wintabs_ui_show_vimtab_name*
```

Comment:
let's remove this option from documentation. We are considering it in code for compatibility reason (old config still works), but we don't need to expose this to new users.